### DISCUSSION TITLE
笔记中无法正常显示表情图标

### DISCUSSION THREAD

**Discussion by ZhangWeilong** [Tokinx/Wing#58] [问答]
![image](https://user-images.githubusercontent.com/8246653/195978633-b56aa9b2-3ad4-4c09-a2e7-d0310b78ff49.png)

测试了些表情，发现表情无法正常显示，都是例如“&#x1f973;”种显示的。
不知道如何处理。

**Comment by Tokinx** [comment]
似乎是你数据库格式选择错误导致，具体设置请参考这篇文章：https://juejin.cn/post/6844903847018627079

**Reply by ZhangWeilong** [reply]
之前只做了MySQL配置、数据库和表的配置修改，并没有做表中字段的编码修改，字段同样修改为utf8mb4后就可以正常显示了。

**Reply by Tokinx** [reply]
可能是不同的数据库版本默认数据库格式不同导致
